js/__internal/grids/grid_core/data_controller/m_data_controller.ts

```diff
@@ -1432,7 +1433,8 @@ export class DataController extends ControllerWithDataMixin {
     }
   }
 
-  getRowIndexOffset() {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  getRowIndexOffset(byLoadedRows?: boolean) {
```

Comment:
Question.

Why do we need the unused variable here?

Reply:
For proper typing in extenders. This method is extended in virtual scrolling with new param, but type is calculated from base class